contracts/contracts/beacon/BeaconProofsLib.sol

```diff
@@ -123,6 +123,7 @@ library BeaconProofsLib {
             withdrawalAddressFromProof == withdrawalAddress,
             "Invalid withdrawal address"
         );
+        require(proof[0] == 0x02, "Invalid validator type");
```

Comment:
That ended up being an easy check. Well done

Reply:
Do you think we should verify that bytes 1 - 11 are of `0x0` value as well?